wpeverest/everest-forms
# Feature - Lookup field

### All Submissions:



### Changes proposed in this Pull Request:

This PR gives the new lookup field which allows a form to “look up” data from another forms. Please check the link to get more detail -> [https://themegrill.atlassian.net/browse/EVF-621](url).

### How to test the changes in this Pull Request:

1. Active the Everest form free.
2. Edit a form.
3. Search the lookup field and verify whether it is there or not.
4. To check functionality-wise active EVF pro.

### PR Link:
[PR](https://github.com/wpeverest/everest-forms-pro/pull/786)

### Types of changes:


### Other information:


### Changelog entry

> Feature - Lookup field.
